DOC-1809: Modify AZ selection configuration (#2301)
RedisLabs/redislabs-docs: content/rc/subscriptions/create-flexible-subscription.md

| **Cloud account** | To deploy this subscription to a specific cloud account, select it here.  Use the Add button to add a new cloud account. |
| **VPC configuration** | Select _In a new VPC_ to deploy to a new Virtual Private Cloud.<br/><br/>To deploy this subscription to an existing Virtual Private Cloud , select _In existing VCP_ and then set VPC ID to the appropriate ID value.   |
| **Deployment CIDR** | The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) subnet address for your deployment. Must not overlap other addresses used with your subscription.|

> | **Allowed Availability Zones** | The availability zone for your selected region.<br/><br/>If you choose *Manual selection*, you must choose at least one zone ID from the **Zone IDs** dropdown.  For more information, see [Availability zones](#availability-zones). |

Reviewer: ```suggestion
| **Allowed Availability Zones** | The availability zones for your selected region.<br/><br/>If you choose *Manual selection*, you must select at least one zone ID from the **Zone IDs** list.  For more information, see [Availability zones](#availability-zones). |
```

When finished, choose **Continue** to determine your subscription size requirements.

{{<image filename="images/rc/button-subscription-continue.png" width="100px" alt="Select the Continue button to continue to the next step." >}}{{< /image >}}

#### Availability zones

> You can reduce network transfer costs by ensuring your Redis Cloud cluster and your application are located in the same availability zone. To specify the availability zone for your hosted cluster, select *Manual Selection* under **Allowed Availability Zones**. 

Reviewer 2: ```suggestion
You can reduce network transfer costs by ensuring your Redis Cloud cluster and your application are located in the same availability zone. 

To specify the availability zone for your hosted cluster, select *Manual Selection* under **Allowed Availability Zones**. 
```

> You can reduce the amount of data transfer between your Redis Cloud cluster and your application by ensuring they are located in the same availability zone. To specify the availability zone for your hosted cluster, select *Manual Selection* under **Allowed Availability Zones**. 

Reviewer: For "reduce the amount of data transfer," does this refer to reducing network latency due to network hops between distant zones?
```suggestion
You can reduce network latency between your Redis Cloud cluster and your application by ensuring they are located in the same availability zone. To specify the availability zone for your hosted cluster, select *Manual Selection* under **Allowed Availability Zones**. 
```

> You can reduce the amount of data transfer between your Redis Cloud cluster and your application by ensuring they are located in the same availability zone. To specify the availability zone for your hosted cluster, select *Manual Selection* under **Allowed Availability Zones**. 

Author: ```suggestion
You can reduce network transfer costs by ensuring your Redis Cloud cluster and your application are located in the same availability zone. To specify the availability zone for your hosted cluster, select *Manual Selection* under **Allowed Availability Zones**. 
```

> If **Multi-AZ** is on, select three availability zones from the **Zone IDs** dropdown.

Reviewer: Quick question: do you have to select exactly 3 availability zones here? Or can you select up to 3?
```suggestion
If **Multi-AZ** is enabled, select three availability zones from the **Zone IDs** list.
```

{{<image filename="images/rc/availability-zones-multi-az.png" width="95%" alt="Select the Continue button to continue to the next step." >}}{{< /image >}}

> Otherwise, select one availability zone from the **Zone IDs** dropdown.

Reviewer: ```suggestion
Otherwise, select one availability zone from the **Zone IDs** list.
```

| **Deployment CIDR** | The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) subnet address for your deployment. Must not overlap other addresses used with your subscription.|
| **Allowed Availability Zones** | The availability zones for your selected region.<br/><br/>If you choose *Manual selection*, you must select at least one zone ID from the **Zone IDs** list.  For more information, see [Availability zones](#availability-zones). |

When finished, choose **Continue** to determine your subscription size requirements.

{{<image filename="images/rc/button-subscription-continue.png" width="100px" alt="Select the Continue button to continue to the next step." >}}{{< /image >}}

#### Availability zones

You can reduce network transfer costs and network latency by ensuring your Redis Cloud cluster and your application are located in the same availability zone. 

To specify the availability zone for your cluster, select *Manual Selection* under **Allowed Availability Zones**. 

> If **Multi-AZ** is enabled, select three availability zones from the **Zone Names** list.

Reviewer: Case doesn't match the screenshot.
```suggestion
If **Multi-AZ** is enabled, select three availability zones from the **Zone names** list.
```

{{<image filename="images/rc/availability-zones-multi-az.png" width="95%" alt="Select Manual selection to select three availability zones when Multi-AZ is enabled." >}}{{< /image >}}

> Otherwise, select one availability zone from the **Zone Names** list.

Reviewer: Case & name don't quite match the screenshot.
```suggestion
Otherwise, select one availability zone from the **Zone name** list.
```

{{<image filename="images/rc/availability-zones-no-multi-az.png" width="90%" alt="Select one availability zone when Multi-AZ is turned off." >}}{{< /image >}}

> For more information on availability zones, see the [GCP docs](https://cloud.google.com/compute/docs/regions-zones/#available) or the [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones).

Reviewer: Maybe consider moving this more info statement to the end of the AZ section?
